### ISSUE TITLE
Training script "labels" too generic & not utilized

### ISSUE THREAD

**Issue by ceholden** [ceholden/yatsm#64]
Function [`get_training_inputs`](https://github.com/ceholden/yatsm/blob/653c1c4718356de3ae1e4f2e9bd05398941a75e7/yatsm/cli/train.py#L184) returns `labels`. These values are meant to be identification values for the pixel in the training data so that groups of pixels sharing the same `label` value can be grouped together for things like K-fold cross-validation. Currently, however, there are two problems:

1. `labels` is too specific. The training data classification label, is a `label`, so this should be more specific to what it actually means
    * Could instead be, for example, `roi_id`, `roi_label`, etc.
2. Nothing becomes of these labels! The [`SpatialKFold`](https://github.com/ceholden/yatsm/blob/653c1c4718356de3ae1e4f2e9bd05398941a75e7/yatsm/classifiers/diagnostics.py#L36) generators both use labels so that individuals from the same label are not mixed in the training / testing splits, but the labels are generated using `scipy.ndimage.label` instead of giving the user the ability to specify these labels via their ROI image.
